helpers/deploymentConfig.ts

```diff
@@ -863,6 +871,38 @@ export const assets: Assets = {
       oracle: "chainlinkFixed",
     },
   ],
+  arbitrumone: [
+    {
+      token: "WBTC",
+      address: "0x2f2a2543b76a4166549f7aab2e75bef0aefc5b0f",
+      price: "63000000000000000000000",
```

Comment:
I would not add the `price` entries into this items, to avoid confusion

Reply:
Should i remove prices from old listed ones too?

Reply:
Removed it from the places where it was not needed.
https://github.com/VenusProtocol/oracle/pull/192/commits/3cd18f5b3be060eb9fcff1ec23a3f29203528d72